src/libraries/System.Runtime.Serialization.Formatters/src/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -1,16 +1,29 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppCurrent)-browser;$(NetCoreAppCurrent)-ios;$(NetCoreAppCurrent)-tvos;$(NetCoreAppCurrent)-android</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum);$(NetCoreAppMinimum)-browser;$(NetCoreAppMinimum)-ios;$(NetCoreAppMinimum)-tvos;$(NetCoreAppMinimum)-android</TargetFrameworks>
     <UseCompilerGeneratedDocXmlFile>false</UseCompilerGeneratedDocXmlFile>
+    <IsPackable>true</IsPackable>
+    <EnablePackageValidation>false</EnablePackageValidation>
   </PropertyGroup>
 
   <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
   <PropertyGroup>
     <TargetPlatformIdentifier Condition="$(TargetFramework.Contains('-'))">$(TargetFramework.SubString($([MSBuild]::Add($(TargetFramework.IndexOf('-')), 1))))</TargetPlatformIdentifier>
-    <!-- TargetsMobile: When we replace implementations with PNSE, need to suppress some "field is never assigned to" warnings. -->
-    <NoWarn Condition="'$(TargetPlatformIdentifier)' != ''">$(NoWarn);CS0649</NoWarn>
     <NoWarn>$(NoWarn);CA1822;IDE0060</NoWarn>
+
+  <!--
+      The real implementation of this library is built for NetCoreAppMinimum,
+      the NetCoreAppCurrent build has a non-functional copy of BinaryFormatter.
+
+      The NetCoreAppCurrent build is only included in the shared runtime,
+      and should always lose to the package for unification; so it is pinned
+      at an assembly version that will always lose.
+  -->
+    <AssemblyVersion Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">8.1.0.0</AssemblyVersion>
+    <IncludeBuildOutput Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">false</IncludeBuildOutput>
+    <SuppressDependenciesWhenPacking Condition="'$(TargetFramework)' == '$(NetCoreAppCurrent)'">true</SuppressDependenciesWhenPacking>
+    <FunctioningBinaryFormatter Condition="'$(TargetFramework)' == '$(NetCoreAppMinimum)'">true</FunctioningBinaryFormatter>
```

Comment:
This looks good. Consider moving this property up and use it for the other conditions.